pkg/components/cluster-autoscaler/component.go

```diff
@@ -118,7 +118,10 @@ type packetConfiguration struct {
 	AuthToken     string
 }
 
-func newComponent() *component {
+// NewConfig returns new Cluster Autoscaler component configuration with default values set.
```

Comment:
We should be consistent with the component name as used in the previous commit.

```suggestion
// NewConfig returns new cluster-autoscaler component configuration with default values set.
```

Reply:
Ditto official name: https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#cluster-autoscaler

Reply:
Fixed.